.github/ISSUE_TEMPLATE/rfc.yml

```diff
@@ -0,0 +1,99 @@
+  - type: input
+    id: relation
+    attributes:
+      label: Is this related to an existing feature request or issue?
+      description: Please share a link, if applicable
+  - type: dropdown
+    id: area
+    attributes:
+      label: Which AWS Lambda Powertools utility does this relate to?
+      options:
+        - Tracer
+        - Logger
+        - Metrics
+        - Parameters
+        - Idempotency
+        - Other
+    validations:
+      required: true
+  - type: textarea
+    id: summary
+    attributes:
+      label: Summary
+      description: Please provide an overview in one or two paragraphs
+    validations:
+      required: true
+  - type: textarea
+    id: problem
+    attributes:
+      label: Use case
+      description: Please share the use case and motivation behind this proposal
+    validations:
+      required: true
+  - type: textarea
+    id: proposal
+    attributes:
+      label: Proposal
+      description: Please explain the design in detail, so anyone familiar with the project could implement it
+      placeholder: What the user experience looks like before and after this design?
+    validations:
+      required: true
+  - type: textarea
+    id: scope
+    attributes:
+      label: Out of scope
+      description: Please explain what should be considered out of scope in your proposal
+    validations:
+      required: true
+  - type: textarea
+    id: challenges
+    attributes:
+      label: Potential challenges
+      description: Nothing is perfect. Please share what common challenges, edge cases, unresolved areas, and suggestions on how to mitigate them
+    validations:
+      required: true
+  - type: textarea
+    id: integrations
+    attributes:
+      label: Dependencies and Integrations
+      description: If applicable, please share whether this feature has additional dependencies, and how it might integrate with other utilities available
+    validations:
+      required: false
+  - type: textarea
+    id: alternatives
+    attributes:
+      label: Alternative solutions
+      description: Please describe what alternative solutions to this use case, if any
+      render: Markdown
+    validations:
+      required: false
+  - type: checkboxes
+    id: acknowledgment
+    attributes:
+      label: Acknowledgment
+      options:
+        - label: This feature request meets [Lambda Powertools Tenets](https://awslabs.github.io/aws-lambda-powertools-typescript/latest/#tenets)
+          required: true
+        - label: Should this be considered in other Lambda Powertools languages? i.e. [Python](https://github.com/awslabs/aws-lambda-powertools-python/), [Java](https://github.com/awslabs/aws-lambda-powertools-java/)
+          required: false
+  - type: markdown
+    attributes:
+      value: |
+        ---
+
+        **Disclaimer**: We value your time and bandwidth. As such, any pull requests created on non-triaged issues might not be successful.
```

Comment:
Same. What about:

```suggestion
        **Disclaimer**: After creating an issue, please wait until it is triaged and confirmed by a maintainer before implementing it. This will reduce amount of rework and the chance that a pull request gets rejected.
```